lib/classification_lifecycle.rb

```diff
@@ -118,6 +119,10 @@ def add_project_live_state
     update_classification_metadata(:live_project, project.live)
   end
 
+  def add_user_groups
+    update_classification_metadata(:user_group_ids, user.non_identity_user_group_ids)
```

Comment:
I thought the client would supply this info to capture the correct context of the classification. I.e. i was classifying subject X from assignment Y in user_group P..